src/Parsers/ExpressionElementParsers.cpp

```diff
@@ -1236,6 +1236,10 @@ bool ParserColumnsTransformers::parseImpl(Pos & pos, ASTPtr & node, Expected & e
     ParserKeyword except("EXCEPT");
     ParserKeyword replace("REPLACE");
     ParserKeyword as("AS");
+    ParserKeyword except_strict("EXCEPTSTRICT");
```

Comment:
we can have an additional keyword `STRICT` here. So the grammar is `except [strict] (...)`. And the following code can be simplified.

Reply:
yes